frontend/src/test/scala/bloop/BaseCompileSpec.scala

```diff
@@ -140,6 +140,38 @@ abstract class BaseCompileSpec extends bloop.testing.BaseSuite {
     }
   }
 
+  test(s"compile scala 3 build incrementally") {
```

Comment:
Does the test fail without the fix?

Reply:
Yes, this is based on reproduction in the Metals repository.